src/testCombiner.ts

```diff
@@ -76,14 +76,14 @@ export function deduplicateTestNames(tests: ITestData[]): ITestData[] {
  * @param {[any]} tests This is an array of tests direct from test gen contains 'test', 'classAnnotations',
  * 'imports' and 'staticImports'
  * @param {string} className The desired name of the class
- * @param {string} testedClasses The list of classes the test covers
+ * @param {string} _testedClasses The list of classes the test covers
  * @param {string} packageName The desired package name of the class
  * @returns {ClangStyle}
  */
 export function genTestClass(
   tests: ITestData[],
   className: string,
-  testedClasses: string,
+  _testedClasses: string,
```

Comment:
Let's make packageName the third parameter, and drop this completely:

```suggestion
  packageName: string,
  legacyPackageName?: string,
) {
  if (legacyPackageName) {
    // Deprecated 4th parameter used
    packageName = legacyPackageName;
  }    
```